lib/lotus/action/callbacks.rb

```diff
@@ -192,7 +192,7 @@ module InstanceMethods
         # @api private
         def call(params)
           _run_before_callbacks(params)
-          super
+          super if location_header.nil? || location_header.empty?
```

Comment:
I agree, I could add to https://github.com/lotus/utils/blob/master/lib/lotus/utils/string.rb